pkg/configurations/configurations.go

```diff
@@ -0,0 +1,157 @@
+package configurations
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/fsnotify/fsnotify"
+
+	"github.com/thoas/go-funk"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+
+	"github.com/spf13/viper"
+)
+
+const defaultInterfacesFilter = "veth*"
+const defaultNodeNetworkStateRefresh = 5
+
+var (
+	log        = logf.Log.WithName("configurations")
+	configPath string
+)
+
+// Takes os environment values
+func init() {
+	configPathTemp, isSet := os.LookupEnv("CONFIG_PATH")
+	if !isSet {
+		panic("CONFIG_PATH is mandatory")
+	}
+	configPath = configPathTemp
+}
+
+// Struct to match config file
+type Config struct {
```

Comment:
This can be private. Please add getters (Public) and setters (private) to interface with users and with the watcher. It should be a little cleaner.